--- 2.6.x/Makefile.old	2008-10-16 19:35:33.000000000 -0400
+++ 2.6.x/Makefile	2008-10-16 19:36:49.000000000 -0400
@@ -47,7 +47,7 @@
 # on first call from remote location we get into this path
 # whilst on second call all is managed by the embedding kernel makefile
 
-KDIR            = /lib/modules/$(shell uname -r)/build
+KDIR            = $(KERNEL_PATH)
 PWD             = $(shell pwd)
 
 fglrx-cfiles    = ${fglrx-c-objs:.o=.c}
