--- fglrx/common/lib/modules/fglrx/build_mod/2.6.x/Makefile.orig	2008-12-16 13:22:19.000000000 -0500
+++ fglrx/common/lib/modules/fglrx/build_mod/2.6.x/Makefile	2008-12-16 13:23:54.000000000 -0500
@@ -75,7 +75,7 @@
 	KVER        = $(shell uname -r)
 endif
 
-KDIR            = /lib/modules/$(KVER)/build
+KDIR            = /usr/src/kernels/$(KVER)
 PWD             = $(shell pwd)
 
 fglrx-cfiles    = ${fglrx-c-objs:.o=.c}
